lms/djangoapps/ccx/utils.py

```diff
@@ -34,6 +35,32 @@
 log = logging.getLogger("edx.ccx")
 
 
+def get_ccx_creation_dict(course):
+    """
+    Return dict of rendering create ccx form.
+
+    Arguments:
+        course (CourseDescriptorWithMixins): An edx course
+
+    Returns:
+        dict: A attribute dict for view rendering
+    """
+    # Translators: ccx_connector is url for CCX Connector application for managing creation of CCXs.s
+    context = {
+        'course': course,
+        'create_ccx_url': reverse('create_ccx', kwargs={'course_id': course.id}),
+        'has_ccx_connector': "true" if hasattr(course, 'ccx_connector') and course.ccx_connector else "false",
+        'use_ccx_con_error_message': _(
+            "You cannot create a CCX based on this course because the Advanced Setting \"CCX Connector URL\" is set "
+            "to {CCXConUrlStart} \"{url}\" {CCXConUrlEnd}. Contact a course admin to give you access."
```

Comment:
I asked you to change this error message in https://github.com/mitocw/edx-platform/pull/198#issuecomment-181997802